tox.ini

```diff
@@ -81,6 +81,8 @@ passenv =
     SF_REGRESS_LOGS
     ; Github Actions provided environmental variables
```

Comment:
@sfc-gh-azhan this here changed.

Reply:
duplicate lines in 84 85

Reply:
why? Such a simple change

Reply:
you just need to move line 84 and 85.

Reply:
Done, removed lines 84 and 84 because they're duplicated.